terminus-op/templates/tdb-statefulset.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ .Release.Name}}-sts-{{.Release.Namespace}}
+spec:
+  selector:
+    matchLabels:
+      app: {{.Release.Name }}-sts-{{.Release.Namespace }}
+  serviceName: {{.Release.Name}}-svc-{{.Release.Namespace}}
+  replicas: {{ .Values.replicaCount}}
+  template:
+    metadata:
+      labels:
+        app: {{.Release.Name }}-sts-{{.Release.Namespace }}
+    spec:
+      containers:
+      - name: terminus-db-{{.Release.Name}}
+        image: {{ .Values.statefulset.image.name}}
+        imagePullPolicy: {{ .Values.statefulset.image.pullPolicy}}
+        ports:
+        - containerPort: {{ .Values.statefulset.port}}
+          name: access-port
+
+        # We need to use Volume mounts if we are mounting secrets to the database instance.
+
+        # volumeMounts:

```

Comment:
If we want to persist storage, we should be able to set the volume mounts dynamically as well.

Reply:
We don't have to dynamically mount something specific to the containers as of now, so I've left it empty!